src/sentry/snuba/entity_subscription.py

```diff
@@ -53,6 +52,7 @@
     EntityKey.Events: "timestamp",
     EntityKey.Sessions: "started",
     EntityKey.Transactions: "finish_ts",
+    EntityKey.GenericMetricsCounters: "timestamp",
```

Comment:
@onewland Are counters provisioned for generic metrics?

Reply:
Just for context, this is needed during the validation of the alert rule. https://github.com/getsentry/sentry/blob/bcc644e7084dcee4c6dd290a551b8db9bdba91f4/src/sentry/incidents/serializers/alert_rule.py#L314